src/main/java/com/fitwsarah/fitwsarah/accountsubdomain/presentationlayer/InvoiceResponseModel.java

```diff
@@ -0,0 +1,17 @@
+package com.fitwsarah.fitwsarah.accountsubdomain.presentationlayer;
+
+import lombok.AccessLevel;
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Value;
+
+@Value
+@Builder
+@AllArgsConstructor(access = AccessLevel.PRIVATE)
+public class InvoiceResponseModel {
+    private String invoiceId;
+    private String accountId;
+    private double amount;
+
+    private double content;
```

Comment:
Could you tell me why is content a double?

Reply:
Sorry, I meant content not amount.

Reply:
My appolgoies, i didnt think of that before, your right it is a bit odd that content is double, i think it made been a typo.